packages/server/src/seed/seed.controller.ts

```diff
@@ -389,12 +390,26 @@ export class SeedController {
       const student = await UserModel.findOneOrFail(body.studentId);
       options['creator'] = student;
     }
-    const question: QuestionModel = await QuestionFactory.create({
-      ...options,
-      ...body.data,
-      createdAt: new Date(),
-    });
-    return question;
+    try {
+      const questionTypes = body.data.questionTypes.map((questionType) => ({
+        cid: questionType.cid,
+        name: questionType.name,
+        color: questionType.color,
+      })) as QuestionTypeModel[];
+      const question: QuestionModel = await QuestionFactory.create({
+        ...options,
+        queueId: body.queueId,
+        text: body.data.text,
+        creatorId: body.studentId,
+        groupable: body.data.groupable,
+        location: body.data.location,
+        questionTypes: questionTypes,
+        createdAt: new Date(),
+      });
+      return question;
```

Comment:
I don't think we need to return anything? Also should we use `.save()` as well here?

Reply:
seeds controller had a bunch of unused code, I have deleted it.